src/final1/subtests/ExportCommandTest.java

```diff
@@ -0,0 +1,208 @@
+	 */
+	@Test
+	public void duplicatesTest() {
+		testAgainstTaskSheet(TASK_SHEET_INPUT_FILE_DUPLICATES);
+	}
+
+	private void testAgainstTaskSheet(String[] input) {
+		// expectedLines are taken from the task sheet, optionalLines not
+		String[] expectedLines = new String[] {
+				"libreoffice -> impress [label=haspart]",
+				"writer -> libreoffice [label=partof]",
+				"centos7 -> operatingsystem [label=containedin]",
+				"centos6 -> operatingsystem [label=containedin]",
+				"centos5 -> operatingsystem [label=containedin]",
+				"calc -> officesuite [label=containedin]",
+				"impress -> libreoffice [label=partof]",
+				"officesuite -> impress [label=contains]",
+				"libreoffice -> writer [label=haspart]",
+				"software -> operatingsystem [label=contains]",
+				"officesuite -> libreoffice [label=contains]",
+				"impress -> officesuite [label=containedin]",
+				"calc -> libreoffice [label=partof]",
+				"software -> officesuite [label=contains]",
+				"centos6 -> centos7 [label=predecessorof]",
+				"officesuite -> calc [label=contains]",
+				"operatingsystem -> centos5 [label=contains]",
+				"centos5 -> centos6 [label=predecessorof]",
+				"operatingsystem -> centos6 [label=contains]",
+				"libreoffice -> calc [label=haspart]",
+				"officesuite -> writer [label=contains]",
+				"libreoffice -> officesuite [label=containedin]",
+				"operatingsystem -> centos7 [label=contains]",
+				"writer -> officesuite [label=containedin]",
+				"centos7 -> centos6 [label=successorof]",
+				"officesuite -> software [label=containedin]",
+				"operatingsystem -> software [label=containedin]",
+				"centos6 -> centos5 [label=successorof]",
+				"software [shape=box]",
+				"officesuite [shape=box]",
+				"operatingsystem [shape=box]"
+		};
+		String[] optionalLines = new String[] {
+				"centos5",
+				"centos6",
+				"centos7",
+				"writer",
+				"calc",
+				"impress",
+				"libreoffice"
+		};
+		testExport(input, expectedLines, optionalLines);
+	}
+
+	private void testExport(String[] input, String[] expectedLines, String[] optionalLines) {
+		List<Matcher<String>> expectedResults = new ArrayList<>();
+		for (String line : expectedLines) {
+			expectedResults.add(leadingSpaces(line));
+		}
+		List<Matcher<String>> optionalResults = new ArrayList<>();
+		for (String line : optionalLines) {
+			optionalResults.add(leadingSpaces(line));
+		}
+
+		runs = new Run[] {
+				new ExportRun("export", expectedResults, optionalResults),
+				new NoOutputRun("quit")
+		};
+		sessionTest(runs, Input.getFile(input));
+	}
+
+	/**
+	 * Returns a matcher that accepts all strings that are {@code text} preceded with an arbitrary number of spaces
+	 * (including none).
+	 */
+	private static Matcher<String> leadingSpaces(final String expectedText) {
+		return new BaseMatcher<String>() {
+
+			@Override
+			public boolean matches(final Object testObject) {
+				String text = (String) testObject;
+				// it's unclear whether other whitespace (\s) is allowed as well
```

Comment:
It's hard to remember all the sources, but I'm quite sure according to the ILIAS `\\s+` is okay.